composer.json

```diff
@@ -4,9 +4,9 @@
     "license": "MIT",
     "description": "A pack for the Doctrine ORM",
     "require": {
-        "composer/package-versions-deprecated: "*",
-        "doctrine/orm": "*",
-        "doctrine/doctrine-bundle": "*",
-        "doctrine/doctrine-migrations-bundle": "*"
+        "composer/package-versions-deprecated": "*",
+        "doctrine/orm": "^2",
+        "doctrine/doctrine-bundle": "^2",
+        "doctrine/doctrine-migrations-bundle": "^2"
```

Comment:
doctrine-migrations-bundle should probably allow v3 as well though

Reply:
Yep, as soon as more ppl + docs have adopted doctrine/migrations v3